modules/rosa-sdpolicy-platform.adoc

```diff
@@ -68,6 +68,24 @@ ifndef::openshift-rosa-hcp[]
 endif::openshift-rosa-hcp[]
 clusters at this time. However, custom labels are supported when creating new machine pools.
 
+[id="rosa-sdpolicy-node-lifecycle_{context}"]
+== Node lifecycle
+
+Worker node persistence is not guaranteed and may be replaced at any time as part of the normal operation and management of OpenShift.
+
+Several events affect the longevity of a {product-title} worker node:
+
+* Machine health checks are deployed and configured to ensure that a worker node with a `NotReady` status is replaced to ensure smooth operation of the cluster.
+* AWS EC2 instance retirement may occur at any time when AWS detects irreparable failure of the underlying hardware that hosts the instance.
```

Comment:
To keep wording consistent, perhaps replacing "retirement" with "longevity" may ease readability